ui/cypress/e2e/onboarding.test.ts

```diff
@@ -0,0 +1,359 @@
+interface TestUser {
+  username: string
+  password: string
+  org: string
+  bucket: string
+}
+
+const defTimeOut = 3000
```

Comment:
Why are you overriding the default four second timeout for the location command?

Reply:
This was an experiment that got left in the test - removed